src/utils.js

```diff
@@ -40,6 +44,34 @@ export const parseWhere = request => {
   return where;
 };
 
+export const parseLimitAndOffset = (request) => {
+  const { limit, offset } = request.query;
+  return {
+    limit: isUndefined(limit) ? void 0 : toNumber(limit),
```

Comment:
Oh, you might be surprised to know that I didn't know it would return `undefined`, I mean I've never seen `void` anywhere before :grin:, is there a problem with `null` here?